services/users-service/src/handlers/createUser.js

```diff
@@ -32,11 +39,33 @@ async function createUser(event, context) {
 
     await dynamoDB.put(params).promise();
 
-    return successResponse({ message: 'User created!' });
+    //OLD
+    // return successResponse({ message: 'User created!' });
+    return successResponse(user);
   } catch (error) {
     console.log(error);
     throw error;
   }
 }
 
 export const handler = commonMiddleware(createUser);
+
+function getStocks(numSelection, companies) {
+  const copy = {...companies};
```

Comment:
First, this is a good habit because it will help you with data immutability.
There are two issues here though:

1. The spread syntax only makes a shallow copy. So all of the nested reference types are not copied, their references are maintained. Not idea for immutability. Better to use something like cloneDeep from lodash when you have nested reference types
2. You're using the Object spread syntax on an Array which does not throw an error but probably isn't what you want